scripts/inference.py

```diff
@@ -133,12 +133,9 @@ def main(args_in: Optional[List[str]] = None) -> None:
     if is_win():
         path = Path(args.build_dir, "./Bin/Release/run_{}.exe".format(model_name))
     else:
-        if args.one_click_run == "True":
-            import neural_speed
-            package_path = os.path.dirname(neural_speed.__file__)
-            path = Path(package_path, "./run_{}".format(model_name))
-        else:
-            path = Path(args.build_dir, "./bin/run_{}".format(model_name))
```

Comment:
Looks good~, but I'm not sure if somewhere else will get into this else branch.

> Tested with same commands provided in the README file, on diverse models requiring token access ID: llama2-7b, llama2-13b, llama2-70b. The script completes its executions as expected.

Could you please provide test commands?

Reply:
@Zhenzhong1 this commit should not be an issue, please refer to PR #92 for more info on why I removed the if/else , we don't need it anymore ;)